src/dsp_tools/commands/xml_validate/prepare_input.py

```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
+from typing import cast
+
+from lxml import etree
+
+from dsp_tools.commands.xml_validate.models.deserialised import LinkValueDeserialised
+from dsp_tools.commands.xml_validate.models.deserialised import ListValueDeserialised
+from dsp_tools.commands.xml_validate.models.deserialised import PermissionsDeserialised
+from dsp_tools.commands.xml_validate.models.deserialised import ProjectDeserialised
+from dsp_tools.commands.xml_validate.models.deserialised import ResourceDeserialised
+from dsp_tools.commands.xml_validate.models.deserialised import RichtextDeserialised
+from dsp_tools.commands.xml_validate.models.deserialised import SimpleTextDeserialised
+from dsp_tools.commands.xml_validate.models.deserialised import ValueDeserialised
+from dsp_tools.commands.xml_validate.models.xml_deserialised import PermissionsXML
+from dsp_tools.commands.xml_validate.models.xml_deserialised import ProjectXML
+from dsp_tools.commands.xml_validate.models.xml_deserialised import ResourceXML
+from dsp_tools.utils.xml_utils import parse_and_clean_xml_file
+from dsp_tools.utils.xml_validation import validate_xml
+
+
+def parse_file(file: Path) -> ProjectXML:
+    """Returns an object which follows the structure of the XML closely"""
+    root = _parse_file_validate_with_schema(file)
+    return _transform_into_xml_deserialised(root)
+
+
+def _parse_file_validate_with_schema(file: Path) -> etree._Element:
+    root = parse_and_clean_xml_file(file)
+    validate_xml(root)
+    return root
```

Comment:
to me, this seems redundant: in this experiment, the XML schema validation should never fail, otherwise we're violationg the experiment setup; and if that's the case, why validate?

Reply:
I added that to make sure that the test data is for sure correct and a schema mismatch will not be the cause. And since I did not have to write this code it did not cause any work. Do you want me to remove it?

Reply:
I leave it up to you - but in the context of an experiment regarding validation, seeing validation that is not relevant to the experiment seems a bit confusing to me. But knowing this now, it's also easy enough to ignore